app/services/get_host_content_service.rb

```diff
@@ -19,9 +18,33 @@ def call
 
 private
 
-  attr_writer :target_content_id
+  attr_accessor :target_content_id, :order
 
   def host_content
-    @host_content ||= Queries::GetEmbeddedContent.new(target_content_id).call
+    @host_content ||= Queries::GetEmbeddedContent.new(target_content_id, order_field:, order_direction:).call
```

Comment:
maybe this is for another PR, but should we get around to renaming `GetEmbeddedContent` to `GetHostContent` as I think that's what it's doing?

Reply:
Yeah, good call - I think I'll leave it for now though